### ISSUE TITLE
Darstellung mehrerer Wallboxen

### ISSUE THREAD

**Issue by ledermann** [solectrus/solectrus#2241]
Mit dem nächsten Update des SENEC-Collectors werden Messwerte für jede Wallbox separat übermittelt:
https://github.com/solectrus/senec-collector/issues/219

Diese sollten angezeigt werden, sofern vorhanden.

**Comment by q2doc** [comment]
Kann man das:

wallbox_charge_power0:

über MQTT auch schon "zweckentfremden" und so eine zweite Wallbox sichtbar machen oder geht es aktuell nur bei dem SENEC-Collector? Also analog zu dem forecast:
MQTT_TOPIC_WALLBOX_1_CHARGE_POWER=
MQTT_TOPIC_WALLBOX_2_CHARGE_POWER=

**Comment by Robbe64** [comment]
Wenn man in EVCC zwei Wallboxen hat wird aktuell auch nur die erste berücksichtigt. Aktuell „verschwindet“ der Strom einfach.

**Comment by ledermann** [comment]
@Robbe64 Ja, derzeitig wird nur eine Wallbox von SOLECTRUS berücksichtigt und angezeigt. Der SENEC-Collector übermittelt zwar die Werte der weiteren Wallboxen, es erfolgt aber noch keine Anzeige. Aufgrund der Arbeiten in #2865 ändert sich da sowieso noch etwas.

In deinem Screenshot sehe ich aber gar keine Wallbox, also auch nicht die Erste. Außerdem sehe ich da die Anzeige "Keine Verbindung". Das sollte nicht sein. Falls es da Probleme gibt, lege bitte ein Issue hier an, dann klären wir das: https://github.com/solectrus/hosting/issues

**Comment by deadrabbit87** [comment]
@ledermann Ich weiß, die Fragen sind müselig, aber gibt es eine Roadmap wann man mit mehreren Ladepunkten rechnen _könnte_?

**Comment by ledermann** [comment]
@deadrabbit87 Die Frage ist völlig verständlich. Da aber SOLECTRUS größtenteils eine One-Man-Show ist, versuche ich meine Ressourcen auf die Dinge zu konzentrieren, die den größten Benefit für alle User bringen. Ich habe dazu kürzlich eine Umfrage gestartet:
https://github.com/orgs/solectrus/discussions/3523

Momentan ist dieses Issue (Support für mehrere Wallboxen) kaum gefragt, daher hat es momentan keine hohe Priorität. Das kann sich aber ändern, je nachdem, wie sich das weitere Feedback so entwickeln. Du kannst das mit beeinflussen ;-)

**Comment by deadrabbit87** [comment]
Sorry, ich nochmal. Gibt es denn eine Möglichkeit in der `.env` die Werte von zwei Wallboxen zu addieren? 

Aktuell funktioniert weder PowerSplitter noch die dazugehörigen Berechnungen, weil die Leistung der zweiten Wallbox einfach "verschwindet". 

Als Workaround wäre es hilfreich, wenn man einfach beide Wallbox addiert und diese dann zusammen in Solectrus dargestellt werden. 

In meinem Fall kommen die Werte von evcc. Leider gibt es dort keinen Datenpunkt, der beide Wallboxen summiert ausgibt.

**Comment by q2doc** [comment]
Ich mache das schon länger indirekt über ioBroker (JavaScript Summe) und pushe es dann per MQTT. Fehlt halt leider die Aufschlüsselung nach Wallbox und die Differenzierung nach Auto. Aber sonst funktioniert es stabil.

**Comment by deadrabbit87** [comment]
Danke - hätt ich auch wieder drauf kommen können. Habs jetzt in HomeAssistant umgesetzt.
